### DISCUSSION TITLE
Changelog available sections

### DISCUSSION THREAD

**Discussion by fugerit79** [olivierlacan/keep-a-changelog#588] [Q&A]
Good evening.

Thanks @olivierlacan  for the great work with these guidelines.

I have a question about sections in changelog.

The types of changes listed are : 

```
Added for new features.
Changed for changes in existing functionality.
Deprecated for soon-to-be removed features.
Removed for now removed features.
Fixed for any bug fixes.
Security in case of vulnerabilities.
```
I have a specific question about how to insert a specific information.
In one project I need to skip a version (becouse the maven deploy failed and that version number is lost).

So in the changelog I would have 

8.2.7

8.2.9

Would you suggest to say some how that the version 8.2.8 has been skipped?

Thanks in advance.

M.

**Comment by olivierlacan** [comment]
Thank you for the kind words. 😊

I think it would make sense to put some information in the changelog about the fact that there was issues with a specific version so that if someone tries to download it or has some issue related to it they can figure it out based on the changelog. 

This is sometimes an issue in the Ruby world, because it’s possible to yank specific be released versions that may have been hacked or have some legal dispute.

**Reply by fugerit79** [reply]
Hello @olivierlacan 

I was wondering if was not the case to introduce some proposal in your keepachangelog guidelines, especially if a version breaks completely previous behaviour.

Something like :

```
## [1.1.0 - 2024-02-05]

### Notice (breaks behaviour | legal issue | build issue ...]

- this version should be skipped becouse.... [link to article/issue/discussion describing the reason]
```

I know keepachangelog can be already customized to people need, bug I would greatly appriciate a proposal about it.

Thanks in advance.

**Comment by fugerit79** [comment]
Ok thanks for the answer. And can you provide an example of how you would do it? (the section and text?)
Thanks again for everything!
